dnf/cli/output.py

```diff
@@ -20,6 +20,7 @@
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import unicode_literals
+from dnf.cli.format import format_date, format_datetime
```

Comment:
Oh really? Do I understand it correctly - you want to use/reference it as
 format_number(x)
and
 dnf.cli.format.format_time(y)
at the same time in the same file?